test/src/apps/prctl/secure_bits.c

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: MPL-2.0
+
+#define _GNU_SOURCE
+
+#include "../test.h"
+#include <sys/prctl.h>
+#include <sys/wait.h>
+#include <linux/capability.h>
+#include <linux/securebits.h>
+#include <unistd.h>
+
+FN_TEST(get_initial_securebits)
+{
+	// Initially, all securebits should be 0
+	TEST_RES(prctl(PR_GET_SECUREBITS), _ret == 0);
+}
+END_TEST()
+
+FN_TEST(set_and_get_keep_caps)
+{
+	int initial_bits;
+
+	initial_bits = CHECK(prctl(PR_GET_SECUREBITS));
```

Comment:
`CHECK` -> `TEST_SUCC`. In `FN_TEST`, we should avoid using `CHECK` if possible.